src/stores/MediaPropertyStore.js

```diff
@@ -27,6 +27,7 @@ import {LocalizeString} from "@/components/common/Misc.jsx";
 import {Slugify} from "@/components/common/Validation.jsx";
 
 import {Migrations, latestVersion} from "@/migrations/MediaPropertyMigrations.js";
+import {mediaCatalogStore, marketplaceStore, permissionSetStore, mediaPropertyStore} from "@/stores/index.js";
```

Comment:
Never import stores globally within other stores - access them via `this.rootStore.<otherStore`